Mathlib/Algebra/Ring/UnitEquivs.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Rodriguez, Johan Commelin
+-/
+import Mathlib.GroupTheory.GroupAction.ConjAct
+import Mathlib.RingTheory.Subring.Basic
+
+/-!
+# Equivalences relating to units
+
+This file makes a couple equivalences that deal with unit groups under various other operations,
+such as centralizers and stabilizers. These are mainly useful to prove Wedderburn's little theorem.
+
+-/
+
+variable {D : Type*}
+
+/-- The stabilizer of `Dˣ` acting on itself by conjugation at `x : Dˣ` is exactly the
+units of the centralizer of `x : D`. -/
+def unitsCentralizerEquiv [Monoid D] (x : Dˣ) :
```

Comment:
I'd use the name `unitsCentralizerEquiv` (or even `unitsCentralizerMulEquiv`) if the equiv went in the other direction. So I think either rename:
```suggestion
def stabilizerConjActUnitsMulEquiv [Monoid D] (x : Dˣ) :
```
or flip the sides.

Reply:
I had to flip this in a weird way - using `MulEquiv.symm`; feel free to try flip it manually and see if you understand what's breaking! (Also note that `map_mul` is no longer as trivial, but that doesn't seem to be the issue...)